tests/test_iconvsr.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import torch
+
+from mmedit.models.backbones.sr_backbones.iconvsr import IconVSR
+
+
+def test_iconvsr():
+    """Test IconVSR."""
+
+    # since IconVSR contains DCN, only gpu is possible.
+    if torch.cuda.is_available():
```

Comment:
We recommand that you should keep unit test for both `cpu` and `cuda` device, because github CI will not cover gpu test.